_posts/2016-01-15-GitHubAnimator.md

```diff
@@ -39,6 +39,6 @@ After playing around with Reveal.js I found out that the code highlight features
 The last part was to make some changes to Reveal.js and Prism.js CSS's to get the desired visual effect. This is an area I'm a bit out of my comfort zone so it took me a while to get something half decent.
 
 ### The End Result
-The code, as it stands now, is still very crude but it's capable of retrieving all commits for a file in a given repository and generate a presentation using Reveal.js. The presentation has a slide for each commit that changed the specified file. The slides use Prism.js to highlight the code.
+The code, as it stands now, is still very crude but it's capable of retrieving all commits for a file in a given repository and generating a presentation with Reveal.js. The presentation has a slide for each commit that changed the specified file. The slides use Prism.js to highlight the code.
 
-There are still a few things hardcoded on the F# code, so it's not a "consumer" ready product, but some developers might be able to use it as it is. You can find the code [here](https://github.com/pedromsantos/GitHubAnimator).
+There are still a few things hardcoded in F#, so it's not a "consumer" ready product, but some developers might be able to use it as is. You can find the code [here](https://github.com/pedromsantos/GitHubAnimator).
```

Comment:
I would use "in the F# code" here, because the things are not hardcoded in the language itself.

I'm fine with the rest of the changes.